cmplibrary/src/debug/java/com/sourcepoint/cmplibrary/exception/LoggerImpl.kt

```diff
@@ -69,7 +69,10 @@ private class LoggerImpl(
         }
         // send data to datadog
         val mediaType = "application/json".toMediaTypeOrNull()
-        val body: RequestBody = RequestBody.create(mediaType, errorMessageManager.build(e))
+        val error = errorMessageManager.build(e)
+        Log.v("DIA-2555", "===== error request =====")
```

Comment:
I believe we should remove the log :)

Reply:
sure, but only after the manual test I'm working on right now

Reply:
Bump for the removal of logs

Reply:
@Nevazhnovu done, thanks for that